### PR TITLE
Catchup Code Review

### PR DESCRIPTION
**Pull request by edgibbs**
Requesting this review so we have a review of all 10,000+ lines of code, that came in before PRs started.

### CODE REVIEW

**Comment by jonbwalker** [comment]
retest this please

File: src/test/java/gov/ca/cwds/cans/rest/resource/CountyResourceTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  public void getAllCounties_success() throws IOException {
+    // when
+    final CountyDto[] actualResult = clientTestRule
+        .withSecurityToken(AUTHORIZED_ACCOUNT_FIXTURE)
+        .target(API.COUNTIES)
+        .request(MediaType.APPLICATION_JSON_TYPE)
+        .get(CountyDto[].class);
+
+    // then
+    assertThat(actualResult.length, is(not(0)));
```
Comment (edgibbs) [inline src/test/java/gov/ca/cwds/cans/rest/resource/CountyResourceTest.java]: Not super explicit on the assertion.

File: src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    final Response postResponse = clientTestRule
+        .withSecurityToken(AUTHORIZED_ACCOUNT_FIXTURE)
+        .target(ASSESSMENTS + SLASH + START)
+        .request(MediaType.APPLICATION_JSON_TYPE)
+        .post(Entity.entity(request, MediaType.APPLICATION_JSON_TYPE));
+    final AssessmentDto assessment = postResponse.readEntity(AssessmentDto.class);
+
+    // then
+    assertThat(postResponse.getStatus(), is(200));
+    assertThat(assessment, is(not(nullValue())));
```
Comment (edgibbs) [inline src/test/java/gov/ca/cwds/cans/rest/resource/AssessmentResourceTest.java]: not a very explicit assertion

**Comment by edgibbs** [comment]
Closing for now, the codebase has evolved and this captures the need to review older code.

**Pull request closed without merging**